src/ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,212 @@
+// @flow
+
+import React, { Component, Children, cloneElement, isValidElement } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '../styles/withStyles';
+import createStyleSheet from '../styles/createStyleSheet';
+import { fade } from '../styles/colorManipulator';
+import common from '../colors/common';
+
+/**
+ * Specification according to material.io
+ * - Have at least three toggle buttons in a group
+ * - Label buttons with text, an icon, or both
+ */
+
+export const styleSheet = createStyleSheet('MuiToggleButton', theme => ({
+  root: {
+    display: 'inline-block',
+    borderRadius: 2,
+    boxShadow: 'none',
```

Comment:
I have the feeling that it can be removed. I haven't seen the property being defined.